hepdata/modules/records/templates/hepdata_records/components/table_list.html

```diff
@@ -124,7 +124,7 @@ <h4 itemprop="name">{{ table.name }}</h4>
                     {% endif %}
 
                     {% if table.description != '.' %}
-                        <p itemprop="description">{{ table.description }}</p>
+                        <p itemprop="description">{{ table.description|safe }}</p>
```

Comment:
I see what you mean but I suggest we use `sanitize_html` with no tags allowed and `strip=True` so that we just remove any HTML elements and leave the plain text, rather than have the table list littered with `&lt;` etc.